http-add-on/templates/rbac.yml

```diff
@@ -46,19 +26,22 @@ metadata:
     httpscaledobjects.http.keda.sh/version: {{ .Values.images.tag | default .Chart.AppVersion }}
     keda.sh/addon: {{ .Chart.Name }}
     app: {{ .Chart.Name }}
-    name: {{ .Chart.Name }}-manager-role
-    app.kubernetes.io/name: {{ .Chart.Name }}-manager-role
+    name: {{ .Chart.Name }}-role
+    app.kubernetes.io/name: {{ .Chart.Name }}-role
     {{- include "keda-addons-http.labels" . | indent 4 }}
-  name: {{ .Chart.Name }}-manager-role
+  name: {{ .Chart.Name }}-role
+rules:
```

Comment:
this introduced a spec error.

Reply:
You are right, this produces error 😢 
Let me review it and check to cover this by test for the future.

Reply:
https://github.com/kedacore/charts/pull/379

Reply:
can we get a chart v0.4.1? I could provide a PR for a release if you are not already up to it

Reply:
I'm on it, but I want to solve the CI before.

Reply:
One question @holygrolli ,
How are you deploying the chart? I'm just trying with helm and even added again the extra line, I can't reproduce it (due to IDK how to validate it in the CI, which also uses helm)

Reply:
I am using the GitOps approach and actually FluxCD. I first found the explanation [here](https://github.com/fluxcd/helm-controller/issues/283#issuecomment-867693048) where I recognized the Helm chart is not correct.

Reply:
okey... it's complicated to reproduce during CI :(

Reply:
The chart is already available @holygrolli , tomorrow we will announce it, but it's already available